ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -94,11 +160,24 @@ private MigrationConfig(final Map<String, String> configs, final String id) {
   }
 
   private static String getServiceId(final Map<String, String> configs) throws MigrationException {
-    final String ksqlServerUrl = configs.get(MigrationConfig.KSQL_SERVER_URL);
+    final String ksqlServerUrl = configs.get(KSQL_SERVER_URL);
     if (ksqlServerUrl == null) {
       throw new MigrationException("Missing required property: " + MigrationConfig.KSQL_SERVER_URL);
     }
 
-    return ServerVersionUtil.getServerInfo(ksqlServerUrl).getKsqlServiceId();
+    final Client ksqlClient = MigrationsUtil.getKsqlClient(
```

Comment:
Where is this client closed? The reason we used to have two versions of `ServerVersionUtil.getServerInfo()` was so that one didn't require client management (creation and deletion) and the other did.

Reply:
Added `ksqlClient.close()` to this function.